### DISCUSSION TITLE
Will user management be in the next update for the self hosted version?

### DISCUSSION THREAD

**Discussion by mforchner** [plausible/analytics#1199] [Self-Hosted Support] [answered]
Hi, we are big fans of the way Plausible is developed and offered. For our company it makes most sense to use the self hosted version and rather pay as a sponsor then to use the hosted version, as we aim to avoid any data management by third parties if possible.

So a critical information for rolling out Plausible to some of our projects is if the next self hosted version will include user management per project, as was recently added to the hosted version. This is a requirement of our teams and processes and shared links will only be fine as an intermediate solution.

According to https://plausible.io/blog/building-open-source#self-hosted-analytics-will-become-a-twice-a-year-long-term-release there will be two updates a year, and according to https://plausible.io/self-hosted-web-analytics they should include the same features as the hosted version.

So my question is just: Is my assumption correct and the next update to the self hosted repo will bring user management, and that this will be in reach within 3-6 months?

**Comment by ukutaht** [comment, marked as answer]
Hey @mforchner 👋

> So my question is just: Is my assumption correct and the next update to the self hosted repo will bring user management, and that this will be in reach within 3-6 months?

Yes, exactly. User management features will be released in `v1.4.0` in the next few months.

**Reply by mforchner** [reply]
Thanks for the fast reply, perfect.
